src/QDMEwald/tests/test_kspace.cpp

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2019 Juelich Supercomputing Centre
```

Comment:
@JensWehner the tests already had the VOTCA header !? But I added JSC, ok?

Reply:
Sure.